config/environments/production.rb

```diff
@@ -76,16 +74,39 @@
   config.active_support.deprecation = :notify
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  # config.log_formatter = ::Logger::Formatter.new
+  config.log_formatter = ::Logger::Formatter.new
 
-  # Do not dump schema after migrations.
-  config.active_record.dump_schema_after_migration = false
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  config.action_mailer.default_url_options = {
-    host: Addressable::URI.parse(Plek.new.external_url_for("transition")).host,
```

Comment:
I don't think Transition sends any email, so why was this here?

Reply:
Very weird! I'm not sure that it does send emails...

Reply:
Oooh, it used to send exception emails via Airbrake before we got Errbit (which itself was Sentry's predecessor).